docs/hopsworksai/aws/instance_profile_permissions.md

```diff
@@ -14,7 +14,11 @@ Replace *BUCKET_NAME* with appropriate S3 bucket name.
         "S3:GetObject",
         "S3:DeleteObject",
         "S3:AbortMultipartUpload",
-        "S3:ListBucketMultipartUploads"
+        "S3:ListBucketMultipartUploads",
+        "S3:PutLifecycleConfiguration",
+        "S3:GetLifecycleConfiguration",
+        "S3:PutBucketVersioning",
+        "S3:GetBucketVersioning"
```

Comment:
One thing we can do for that is to add a section in `Limiting AWS permissions` that says that you can remove these permissions if you don't activate the backups.

Reply:
updated docs